events/dispatcher_usage.go

```diff
@@ -116,7 +116,7 @@ func (self *usageEventAdapter) AcceptMetricsMsg(message *metrics_pb.MetricsMessa
 				for usageType, usage := range bucket.Values {
 					evt := &event.UsageEvent{
 						Namespace:        event.UsageEventsNs,
-						Version:          event.UsageEventsVersion,
+						Version:          2,
```

Comment:
magic number?

Reply:
Usually have a constant for this, but we're supporting two versions at the same time, so one gets the constant and the other is hard coded to the old version. At point we'll remove support for the old version.